__test__/slackify-markdown.test.js

```diff
@@ -34,7 +34,7 @@ test('Italic', () => {
 
 test('Bold+Italic', () => {
   const mrkdown = '***bold+italic***';
-  const slack = `${zws}*${zws}_bold+italic_${zws}*${zws}\n`;
+  const slack = `${zws}_${zws}*bold+italic*${zws}_${zws}\n`;
```

Comment:
The order has changed here, but it doesn't make a difference: https://app.slack.com/block-kit-builder/#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22_*this*_%20is%20*_fine_*%22%7D%7D%5D%7D